e2e/nx/src/import.test.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  checkFilesExist,
+  cleanupProject,
+  getSelectedPackageManager,
+  newProject,
+  runCLI,
+  updateJson,
+  updateFile,
+} from '@nx/e2e/utils';
+
+describe('Nx Import', () => {
+  let proj: string;
+  beforeAll(
+    () =>
+      (proj = newProject({
+        packages: ['@nx/js'],
+        unsetProjectNameAndRootFormat: false,
+      }))
+  );
+  afterAll(() => cleanupProject());
+
+  it('should be able to import a vite app', () => {
+    const remote = 'https://github.com/nrwl/nx.git';
```

Comment:
Maybe use a local remote so it's not necessary to have a reference on remote?

Reply:
Done